src/bandersnatch/mirror.py

```diff
@@ -82,6 +82,7 @@ def __init__(
         self.homedir = self.storage_backend.PATH_BACKEND(homedir)
         self.lockfile_path = self.homedir / ".lock"
         self.master = master
+        self.filters = LoadedFilters(True)
```

Comment:
```suggestion
        self.filters = LoadedFilters(load_all=True)
```
Maybe just so people know what the bool does without having to go back to filter.py like I did

I've never understood this whole plugin complexity, if we always load all the modules, why do we both having all this extra code? Or am I missing something here?

Reply:
Well currently the plugin groups are loaded when they are first about to be used and then stored in a global variable. However, Mirror loads all of the filter groups anyway so I think it makes sense to just load all of them for the mirror class right away, but having the option to load only the groups they need is useful I think for others using the API.